src/_cffi_src/openssl/ssl.py

```diff
@@ -806,4 +807,8 @@
 static const long Cryptography_HAS_CIPHER_DETAILS = 1;
 #endif
 
+#if CRYPTOGRAPHY_OPENSSL_LESS_THAN_111
```

Comment:
This attribute needs to be available only when it's really present in OpenSSL. See #4476 for all the places that need editing.